Remove dependency on NuGet.CommandLine from C# build

**Describe the bug**

This library has a critical vulnerability, and should not be included in the release build. We should also check to see if we need vswhere.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Type '....'
3. ....
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Sample input/output**
As applicable, add examples of text input and json output to help explain the problem.

**Platform (please complete the following information):**
 - Platform: [e.g. .NET, JavaScript, Python, Java]
 - Environment: [e.g. nuget package, npm package, LUIS, console app, in browser]
 - Version of package [e.g. v1.0.8.1]

**Additional context**
Add any other context about the problem here.

This is still an issue

```warning NU1904: Package 'NuGet.CommandLine' 5.11.5 has a known critical severity vulnerability, https://github.com/advisories/GHSA-68w7-72jg-6qpp```

--- a/.NET/Microsoft.Recognizers.Text/Microsoft.Recognizers.Text.csproj
+++ b/.NET/Microsoft.Recognizers.Text/Microsoft.Recognizers.Text.csproj
@@ -44,7 +44,6 @@
     <PackageReference Include="System.Collections.Immutable" Version="1.7.1" />
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
     <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="2.2.0" />
-    <PackageReference Include="NuGet.CommandLine" Version="5.11.5" />
     <PackageReference Include="vswhere" Version="2.8.4" />
   </ItemGroup>
 

--- a/.NET/Samples/RecognizerFunction/RecognizerFunction.csproj
+++ b/.NET/Samples/RecognizerFunction/RecognizerFunction.csproj
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <AzureFunctionsVersion>v3</AzureFunctionsVersion>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
   </PropertyGroup>

--- a/.NET/Samples/RecognizerFunction/RecognizerFunction.csproj
+++ b/.NET/Samples/RecognizerFunction/RecognizerFunction.csproj
@@ -10,9 +10,8 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="3.0.9" />
+    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="4.6.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-    <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.1.22" />
   </ItemGroup>
   <ItemGroup>
     <Reference Include="Microsoft.CSharp" />

